### PR TITLE
Add unassigned contributions to get Contributions range Resolver

### PR DESCRIPTION
**Pull request by amrro**
## Linear Ticket

[PRO-398 - Add unassigned contributions to the get contribution count range resolver](https://linear.app/govrn/issue/PRO-398/add-unassigned-contributions-to-the-get-contribution-count-range)

## Description
Include unassigned - that belong to no guild - Contribution in get contributions range count Resolver.

### CODE REVIEW

File: apps/protocol-api/src/prisma/resolvers/Contribution.ts [hunk 1]
```diff
@@ -535,33 +533,24 @@ export class ContributionCustomResolver {
...
+             date(date_of_engagement),
+             count(date_of_engagement)      as count
+      FROM "Contribution" c
+             LEFT JOIN "GuildContribution" as gc
+                       ON c."id" = gc."contribution_id"
+             LEFT JOIN "Guild" as g
+                       ON g."id" = gc."guild_id"
+      WHERE (c."date_of_engagement" BETWEEN ${start} AND ${end}
+        AND c."user_id" = ${user_id}
+        AND (NOT c.id IN (SELECT contribution_id as id FROM "GuildContribution") OR gc."guild_id" in (${Prisma.join(
```
Comment (Flip-Liquid) [inline apps/protocol-api/src/prisma/resolvers/Contribution.ts:550]: @amrro so this clause now catches contributions that are not in the guild contribution table?

Comment (amrro) [reply apps/protocol-api/src/prisma/resolvers/Contribution.ts:550]: Yes. It will load unassigned contributions alongside guilds' ones iff provided.

**Review by jonathanprozzi — COMMENTED** [review]
This looks good -- testing by making sure everything is good with the Bar Chart but I'm seeing the 'Unassigned' show up!

**Review by jonathanprozzi — APPROVED** [review]
Works for the chart -- awesome! I can wrap up the remaining user story on that

**Pull request merged**